src/woo_publications/publications/tests/test_publications_api.py

```diff
@@ -399,9 +427,64 @@ def test_create_publication(self):
                 response_data["informatieCategorieen"], [_("This field is required.")]
             )
 
+        with self.subTest("deactivated organisation cannot be used as an organisation"):
+            data = {
+                "publisher": str(deactivated_organisation.uuid),
+                "verantwoordelijke": str(deactivated_organisation.uuid),
+                "opsteller": str(deactivated_organisation.uuid),
+            }
+
+            response = self.client.post(url, data, headers=AUDIT_HEADERS)
+
+            self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+            response_data = response.json()
+
+            # format the same way drf does for gettext to translate the error message properly
```

Comment:
Maybe we should think about setting the test language to english so we won't run into these problems, that way we can just validate on english texts. If we want to go that route then we should create an issue for it. Because finding out how we can and cant translate these 3rd party translatables is annoying.

Reply:
`@override_settings(LANGUAGE_CODE="en")` is totally acceptable for me

Reply:
Yes I might play around with this later because I want to test these translations constantly. And not use it gettext in some tests and overriding the setting in others.